docs/08-guides.md

```diff
@@ -133,6 +133,52 @@ return html`<div id="foo" foo=${40 + 2}>Hello!</div>`
 // ✔ snowpack installed: preact, htm. [1.06s]
 ```
 
+### Lit-element + lit-html
+
+[Lit-element](https://lit-element.polymer-project.org/) is a simple base class for creating fast, lightweight web components that leverages [lit-html](https://lit-html.polymer-project.org/) for fast, efficient and expressive templating. Similar to HTM, lit-html uses tagged template literals which allows for JSX-like syntax in the browser without requiring transpilation.
+
+```js
+// File: src/index.js
+import { LitElement, html, css } from "/web_modules/lit-element.js";
+
+// $ snowpack --include "src/index.js"
+// ✔ snowpack installed: lit-html, lit-element [0.25s]
```

Comment:
I think this would only scan and install `lit-element`.  Or, if it's common to use the two together, there's no harm in adding both `import` statements to the page.